### PR TITLE
Extensions 2021

### PR DESCRIPTION
**Pull request by bobdenotter**

### CODE REVIEW

File: public/theme/extensions-2021/css/bulma.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
...ination-previous:focus,.select select.is-active,.select select.is-focused,.select select:active,.select select:focus,.textarea:active,.textarea:focus{outline:none}.boltform button[disabled],.boltform fieldset[disabled] button,.boltform fieldset[disabled] input,.boltform fieldset[disabled] textarea,.boltform input[disabled],.boltform select fieldset[disabled] select,.boltform select select[disabled],.boltform textarea[disabled],.button[disabled],.file-cta[disabled],.file-name[disabled],.input[disabled],.pagination-ellipsis[disabled],.pagination-link[disabled],.pagination-next[disabled],.pagination-previous[disabled],.select fieldset[disabled] select,.select select[disabled],.textarea[disabled],fieldset[disabled] .boltform button,fieldset[disabled] .boltform input,fieldset[disabled] .boltform select select,fieldset[disabled] .boltform textarea,fieldset[disabled] .button,fieldset[disabled] .file-cta,fieldset[disabled] .file-name,fieldset[disabled] .input,fieldset[disabled] .pagination-ellipsis,fieldset[disabled] .pagination-link,fieldset[disabled] .pagination-next,fieldset[disabled] .pagination-previous,fieldset[disabled] .select select,fieldset[disabled] .textarea{cursor:not-allowed}
```
Comment (I-Valchev) [inline public/theme/extensions-2021/css/bulma.css:1]: I don't think we use Bulma in this project, so is this file needed?

File: public/theme/extensions-2021/img/Twitter_Logo_blue.svg [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (I-Valchev) [inline public/theme/extensions-2021/img/Twitter_Logo_blue.svg:1]: Is this file used somewhere?

File: public/theme/extensions-2021/img/easy.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...953489c-.8790697 0-1.5697674.6906976-1.5697674 1.5697674s.6906977 1.5697675 1.5697674 1.5697675h31.3953489c.8790697 0 1.5697674-.6906977 1.5697674-1.5697675s-.6906977-1.5697674-1.5697674-1.5697674z"/><path d="m52.1162791 41.127907h-31.3953489c-.8790697 0-1.5697674.6906977-1.5697674 1.5697674 0 .8790698.6906977 1.5697675 1.5697674 1.5697675h31.3953489c.8790697 0 1.5697674-.6906977 1.5697674-1.5697675 0-.8790697-.6906977-1.5697674-1.5697674-1.5697674z"/><path d="m20.7209302 12.872093h18.8372093c.8790698 0 1.5697675-.6906977 1.5697675-1.5697674 0-.8790698-.6906977-1.56976746-1.5697675-1.56976746h-18.8372093c-.8790697 0-1.5697674.69069766-1.5697674 1.56976746 0 .8790697.6906977 1.5697674 1.5697674 1.5697674z"/><path d="m20.7209302 31.7093023h18.8372093c.8790698 0 1.5697675-.6906976 1.5697675-1.5697674s-.6906977-1.5697675-1.5697675-1.5697675h-18.8372093c-.8790697 0-1.5697674.6906977-1.5697674 1.5697675s.6906977 1.5697674 1.5697674 1.5697674z"/><path d="m39.5581395 47.4069767h-18.8372093c-.8790697 0-1.5697674.6906977-1.5697674 1.5697675s.6906977 1.5697674 1.5697674 1.5697674h18.8372093c.8790698 0 1.5697675-.6906976 1.5697675-1.5697674s-.6906977-1.5697675-1.5697675-1.5697675z"/></g></svg>
```
Comment (I-Valchev) [inline public/theme/extensions-2021/img/easy.svg:1]: Same for this, is it used?

File: public/theme/extensions-2021/img/meet-up.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...2.5-1.1192881-2.5-2.5v-15.51c-.0149484-3.0252409-2.4259518-5.4937545-5.45-5.58z" fill="#fe3bb5"/><path d="m10.58 12.86c5.8431727 0 10.58 4.7368273 10.58 10.58s-4.7368273 10.58-10.58 10.58c-5.84088762-.0055129-10.57448712-4.7391124-10.58-10.58 0-5.8431727 4.73682735-10.58 10.58-10.58zm0 5c-3.0794662.005504-5.57449603 2.5005338-5.58 5.58 0 3.0817489 2.4982511 5.58 5.58 5.58s5.58-2.4982511 5.58-5.58-2.4982511-5.58-5.58-5.58z" fill="#fe3bb5"/><path d="m58.66 42.67v-6.46c-.0026402-1.8865129-.7559625-3.6944141-2.093677-5.0246239-1.3377144-1.3302097-3.1498251-2.0733491-5.036323-2.0653761h-9.09c-3.9134151.005508-7.084492 3.1765849-7.09 7.09v6.46c0 1.3807119-1.1192881 2.5-2.5 2.5s-2.5-1.1192881-2.5-2.5v-6.46c.0110108-6.6725565 5.4174435-12.0789892 12.09-12.0900166h9.09c3.211773-.0052921 6.2941086 1.2654907 8.5689258 3.5328065s3.5557742 5.3454371 3.5610742 8.5572101v6.46c0 1.3807119-1.1192881 2.5-2.5 2.5s-2.5-1.1192881-2.5-2.5z" fill="#0376fc"/><path d="m65.77 54.96h-16.24v39.54c0 1.3807119-1.1192881 2.5-2.5 2.5s-2.5-1.1192881-2.5-2.5v-39.54h-16.29c-1.3807119 0-2.5-1.1192881-2.5-2.5s1.1192881-2.5 2.5-2.5h37.53c1.3807119 0 2.5 1.1192881 2.5 2.5s-1.1192881 2.5-2.5 2.5z" fill="#000"/></g></svg>
```
Comment (I-Valchev) [inline public/theme/extensions-2021/img/meet-up.svg:1]: Same, is it used?

File: public/theme/extensions-2021/img/open-source.svg [hunk 5]
```diff
@@ -0,0 +1 @@
...0967033 10.3087912s-.6989011 7.0472527-2.0967033 10.3087912c-1.3395604 3.145055-3.2615385 5.9989011-5.6494505 8.445055-2.4461539 2.4461538-5.2417583 4.3681318-8.445055 5.6494505-3.2615385 1.3978022-6.7560439 2.0967033-10.3087912 2.0967033zm0-47.58351648c-5.6494505 0-10.9494505 2.21318681-14.9098901 6.17362638-3.96043957 3.9604396-6.17362638 9.2604396-6.17362638 14.9098901s2.21318681 10.9494505 6.17362638 14.9098901c3.9604396 3.9604396 9.2604396 6.1736264 14.9098901 6.1736264s10.9494505-2.2131868 14.9098901-6.1736264 6.1736264-9.2604396 6.1736264-14.9098901-2.2131868-10.9494505-6.1736264-14.9098901c-3.9604396-3.96043957-9.2604396-6.17362638-14.9098901-6.17362638z"/><path d="m11 22.564029h7.2520345c1.9021729-4.1133657 6.8359341-5.9349991 10.9969375-4.0546034 4.1610033 1.8803958 6.0037334 6.7576724 4.1015604 10.8710381-1.9021729 4.1133658-6.8359341 5.9349992-10.9969375 4.0546034-1.7832871-.8226732-3.2693598-2.23297-4.1015604-4.0546034h-7.2520345c1.902173 8.1092068 10.1052939 13.1040081 18.248972 11.2236123 8.143678-1.8803958 13.2557679-9.9308402 11.3535949-18.040047s-10.1052939-13.04524567-18.248972-11.1648499c-5.647076 1.2927721-10.045851 5.6411873-11.3535949 11.1648499z"/></g></svg>
```
Comment (I-Valchev) [inline public/theme/extensions-2021/img/open-source.svg:1]: Same, is it used?

File: public/theme/extensions-2021/img/simple.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg height="54" viewBox="0 0 53 54" width="53" xmlns="http://www.w3.org/2000/svg"><g fill="#0278fc"><path d="m42.55 15.75c-.9-.9-2.325-.9-3.15 0-.9.9-.9 2.325 0 3.15l8.1 8.1-8.1 8.1c-.9.9-.9 2.325 0 3.15.45.45 1.05.675 1.575.675s1.125-.225 1.575-.675l9.675-9.675c.9-.9.9-2.325 0-3.15z"/><path d="m35 .6c-1.2-.375-2.475.375-2.775 1.575l-13.8 48.45c-.375 1.2.375 2.475 1.575 2.775.225.075.375.075.6.075.975 0 1.875-.675 2.175-1.65l13.725-48.45c.375-1.2-.3-2.475-1.5-2.775z"/><path d="m14.1 15.75c-.9-.9-2.325-.9-3.15 0l-9.675 9.675c-.9.9-.9 2.325 0 3.15l9.675 9.675c.45.45 1.05.675 1.575.675s1.125-.225 1.575-.675c.9-.9.9-2.325 0-3.15l-8.1-8.1 8.1-8.1c.9-.825.9-2.25 0-3.15z"/></g></svg>
```
Comment (I-Valchev) [inline public/theme/extensions-2021/img/simple.svg:1]: Same, is it used?

File: public/theme/extensions-2021/img/slack.svg [hunk 7]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (I-Valchev) [inline public/theme/extensions-2021/img/slack.svg:1]: Same, is it used?

File: public/theme/extensions-2021/index.twig [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+            {% setcontent records = 'packages' order '-time_updated' limit 6 %}
+            {% for record in records %}
+                <div class="news-wrapper">
+                    {% include('partials/_sub_listing_package.twig') %}
+                </div>
+            {% else %}
+                <p>{{ __("general.phrase.no-content-found") }}</p>
+            {% endfor %}
+        </div>
+
```
Comment (I-Valchev) [inline public/theme/extensions-2021/index.twig:27]: Since we don't show all the extensions here, let's add 2 call to action buttons for extensions and themes.

File: public/theme/extensions-2021/js/vendor/baguetteBox.min.js [hunk 9]
```diff
@@ -0,0 +1,279 @@
+/*!
```
Comment (I-Valchev) [inline public/theme/extensions-2021/js/vendor/baguetteBox.min.js:1]: Do we use this?

File: public/theme/extensions-2021/js/vendor/carousel-slider.js [hunk 10]
```diff
@@ -0,0 +1,1165 @@
+var tns = function () {
```
Comment (I-Valchev) [inline public/theme/extensions-2021/js/vendor/carousel-slider.js:1]: Also, is this used?

File: public/theme/extensions-2021/js/vendor/image-scroll.js [hunk 11]
```diff
@@ -0,0 +1,238 @@
+/**
```
Comment (I-Valchev) [inline public/theme/extensions-2021/js/vendor/image-scroll.js:1]: As far as I know, this is only used on the showcases page. Let's remove it.

File: public/theme/extensions-2021/js/vendor/imagesloaded.pkgd.min.js [hunk 12]
```diff
@@ -0,0 +1,156 @@
+/*!
```
Comment (I-Valchev) [inline public/theme/extensions-2021/js/vendor/imagesloaded.pkgd.min.js:1]: Also this?

File: public/theme/extensions-2021/js/vendor/nocost-module.js [hunk 13]
```diff
@@ -0,0 +1,11 @@
+import {annotate} from 'https://unpkg.com/rough-notation?module';
```
Comment (I-Valchev) [inline public/theme/extensions-2021/js/vendor/nocost-module.js:1]: Is this used?

File: public/theme/extensions-2021/js/vendor/slider-module.js [hunk 14]
```diff
@@ -0,0 +1,22 @@
+type = "module"
```
Comment (I-Valchev) [inline public/theme/extensions-2021/js/vendor/slider-module.js:1]: Not sure if this is used either.

File: public/theme/extensions-2021/partials/_footer.twig [hunk 15]
```diff
@@ -0,0 +1,30 @@
+<footer>
+    <nav class="footer-nav">
+        <p>
+            &copy; {{ "now"|date("Y") }} &bullet;
+            {{ __('general.phrase.built-with-bolt', {'%url%':'https://bolt.cm'}) }}
+        </p>
+{#        {% for item in menu_array('footer') %}#}
```
Comment (I-Valchev) [inline public/theme/extensions-2021/partials/_footer.twig:7]: Comments! 🙈

File: public/theme/extensions-2021/partials/_fresh_install.twig [hunk 16]
```diff
@@ -0,0 +1,13 @@
+{% if user %}
```
Comment (I-Valchev) [inline public/theme/extensions-2021/partials/_fresh_install.twig:1]: We don't need this.

File: public/theme/extensions-2021/partials/_header.twig [hunk 17]
```diff
@@ -0,0 +1,46 @@
...
+                                            <li>
+                                                <a href="{{ subitem.link }}">{{ subitem.label }}</a>
+                                            </li>
+                                        {% endfor %}
+                                    </ul>
+                                </div>
+                            {% endif %}
+                        </li>
+                    {% endfor %}
+                    <div class="search-container">
```
Comment (I-Valchev) [inline public/theme/extensions-2021/partials/_header.twig:34]: As discussed, let's make sure that the search field here and on the search page uses the styles that Derk made for https://boltcms.io/submit-showcase

File: public/theme/extensions-2021/partials/_master.twig [hunk 18]
```diff
@@ -0,0 +1,72 @@
...
+
+    <link href="https://fonts.googleapis.com/css2?family=Work+Sans:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+          rel="stylesheet">
+
+    <link rel="stylesheet" type="text/css" href="{{ asset('css/app.css', 'theme') }}">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="shortcut icon" href="{{ asset('img/favicon.ico', 'theme') }}">
+
+    <!-- Matomo -->
```
Comment (I-Valchev) [inline public/theme/extensions-2021/partials/_master.twig:16]: Can you confirm if this is the right code for the extensions page?

Each site/subsite gets its own Matomo tracking code 👍

File: public/theme/extensions-2021/partials/_navbar.twig [hunk 19]
```diff
@@ -0,0 +1,36 @@
+{# Navbar #}
+<nav class="navbar is-fixed-top is-primary" role="navigation" aria-label="main navigation">
+    <div class="container">
+        <div class="navbar-brand">
+{#            <span class="navbar-item">#}
```
Comment (I-Valchev) [inline public/theme/extensions-2021/partials/_navbar.twig:5]: Comments 🙈

File: public/theme/extensions-2021/partials/_navbar.twig [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+                <span></span>
+                <span></span>
+            </span>
+        </div>
+        <div class="navbar-menu" id="navbar-toggle">
+            <div class="navbar-start">
+                {{ menu(template = 'partials/_sub_menu.twig') }}
+            </div>
+            <div class="navbar-end">
+{#                <div class="select ">#}
```
Comment (I-Valchev) [inline public/theme/extensions-2021/partials/_navbar.twig:19]: 🙈

File: public/theme/extensions-2021/css/bulma.css (hunk 1 above)
Comment (mcdennem) [reply public/theme/extensions-2021/css/bulma.css:1]: No, deleted.

File: public/theme/extensions-2021/img/Twitter_Logo_blue.svg (hunk 2 above)
Comment (mcdennem) [reply public/theme/extensions-2021/img/Twitter_Logo_blue.svg:1]: No, deleted.

File: public/theme/extensions-2021/img/easy.svg (hunk 3 above)
Comment (mcdennem) [reply public/theme/extensions-2021/img/easy.svg:1]: No, deleted.

File: public/theme/extensions-2021/img/meet-up.svg (hunk 4 above)
Comment (mcdennem) [reply public/theme/extensions-2021/img/meet-up.svg:1]: No, deleted.

File: public/theme/extensions-2021/img/open-source.svg (hunk 5 above)
Comment (mcdennem) [reply public/theme/extensions-2021/img/open-source.svg:1]: No, deleted.

File: public/theme/extensions-2021/partials/_sub_recent_packages.twig [hunk 21]
```diff
@@ -0,0 +1,24 @@
+{# The next section iterates over all of the contenttypes, and prints a list
+    of the five latest records of each of them. The 'magic' happens in the
+    setcontent tag.. ct.slug ~ "/latest/3" concatenates the slug of the
+    contenttype with /latest/3, so it becomes something like "pages/latest/3".
+#}
+
+            {% for type in ['bolt-extension', 'bolt-theme'] %}
```
Comment (I-Valchev) [inline public/theme/extensions-2021/partials/_sub_recent_packages.twig:7]: Let's format this properly

File: public/theme/extensions-2021/img/simple.svg (hunk 6 above)
Comment (mcdennem) [reply public/theme/extensions-2021/img/simple.svg:1]: No, deleted.

File: public/theme/extensions-2021/img/slack.svg (hunk 7 above)
Comment (mcdennem) [reply public/theme/extensions-2021/img/slack.svg:1]: No, deleted.

File: public/theme/extensions-2021/source/gulpfile.js [hunk 22]
```diff
@@ -0,0 +1,81 @@
+var gulp = require('gulp');
```
Comment (I-Valchev) [inline public/theme/extensions-2021/source/gulpfile.js:1]: Pretty sure we don't need gulp here :-)

File: public/theme/extensions-2021/index.twig (hunk 8 above)
Comment (mcdennem) [reply public/theme/extensions-2021/index.twig:27]: ok

File: public/theme/extensions-2021/js/vendor/baguetteBox.min.js (hunk 9 above)
Comment (mcdennem) [reply public/theme/extensions-2021/js/vendor/baguetteBox.min.js:1]: No, deleted.

File: public/theme/extensions-2021/source/scss/_breakpointdebug.scss [hunk 23]
```diff
@@ -0,0 +1,33 @@
+@if ($debug-breakpoint==true) {
```
Comment (I-Valchev) [inline public/theme/extensions-2021/source/scss/_breakpointdebug.scss:1]: This can be removed

File: public/theme/extensions-2021/js/vendor/carousel-slider.js (hunk 10 above)
Comment (mcdennem) [reply public/theme/extensions-2021/js/vendor/carousel-slider.js:1]: No, deleted.

File: public/theme/extensions-2021/source/scss/_settings.scss [hunk 24]
```diff
@@ -0,0 +1,80 @@
+// Show the current breakpoints in the lower left corner.
```
Comment (I-Valchev) [inline public/theme/extensions-2021/source/scss/_settings.scss:1]: This too.

File: public/theme/extensions-2021/js/vendor/image-scroll.js (hunk 11 above)
Comment (mcdennem) [reply public/theme/extensions-2021/js/vendor/image-scroll.js:1]: No, deleted.

File: public/theme/extensions-2021/js/vendor/imagesloaded.pkgd.min.js (hunk 12 above)
Comment (mcdennem) [reply public/theme/extensions-2021/js/vendor/imagesloaded.pkgd.min.js:1]: No, deleted.

File: public/theme/extensions-2021/js/vendor/nocost-module.js (hunk 13 above)
Comment (mcdennem) [reply public/theme/extensions-2021/js/vendor/nocost-module.js:1]: No, deleted.

File: public/theme/extensions-2021/js/vendor/slider-module.js (hunk 14 above)
Comment (mcdennem) [reply public/theme/extensions-2021/js/vendor/slider-module.js:1]: No, deleted.

File: public/theme/extensions-2021/source/scss/_typography.scss [hunk 25]
```diff
@@ -0,0 +1,8 @@
+h1, h2, h3, h4, h5, h6 {
```
Comment (I-Valchev) [inline public/theme/extensions-2021/source/scss/_typography.scss:1]: (OK, let's say that all the scss files can go)

File: public/theme/extensions-2021/partials/_footer.twig (hunk 15 above)
Comment (mcdennem) [reply public/theme/extensions-2021/partials/_footer.twig:7]: Yes, the footer is empty. Maybe we can fill it like the Bolt website or leave it like it is.

File: public/theme/extensions-2021/partials/_fresh_install.twig (hunk 16 above)
Comment (mcdennem) [reply public/theme/extensions-2021/partials/_fresh_install.twig:1]: deleted.

File: public/theme/extensions-2021/partials/_footer.twig (hunk 15 above)
Comment (I-Valchev) [reply public/theme/extensions-2021/partials/_footer.twig:7]: ah, good point! Ok, let's leave it as-is for now, we can always uncomment it later if we add a menu! 👍

File: public/theme/extensions-2021/source/scss/_typography.scss (hunk 25 above)
Comment (mcdennem) [reply public/theme/extensions-2021/source/scss/_typography.scss:1]: Ok, deleted

File: public/theme/extensions-2021/partials/_sub_recent_packages.twig (hunk 21 above)
Comment (mcdennem) [reply public/theme/extensions-2021/partials/_sub_recent_packages.twig:7]: Done

File: public/theme/extensions-2021/partials/_navbar.twig (hunk 19 above)
Comment (mcdennem) [reply public/theme/extensions-2021/partials/_navbar.twig:5]: Whole navbar.twig is 🔥

File: public/theme/extensions-2021/partials/_navbar.twig (hunk 20 above)
Comment (mcdennem) [reply public/theme/extensions-2021/partials/_navbar.twig:19]: Whole navbar.twig is 🔥

File: public/theme/extensions-2021/partials/_master.twig (hunk 18 above)
Comment (mcdennem) [reply public/theme/extensions-2021/partials/_master.twig:16]: No, I have deleted it

File: public/theme/extensions-2021/partials/_header.twig (hunk 17 above)
Comment (mcdennem) [reply public/theme/extensions-2021/partials/_header.twig:34]: 😎
